src/Jawa/Event/Time.elm

```diff
@@ -0,0 +1,48 @@
+module Jawa.Event.Time exposing (Time, decoder, encoder)
+
+{-|
+
+@docs Time, decoder, encoder
+
+-}
+
+import Jawa.SeekRange as SR
+import Jawa.Viewable as V
+import Json.Decode
+import Json.Encode
+
+
+{-| <https://docs.jwplayer.com/players/reference/seek-events-1#ontime>
+-}
+type alias Time =
+    { currentTime : Float
+    , duration : Float
+    , position : Float
```

Comment:
This also seems to include an undocumented `metadata` field that has a duplicate of the `currentTime` value, like this:

``` json
{
  "metadata": {
    "currentTime": 12.34
  }
}
```

On the one hand, it doesn't seem worthwhile to both parsing that out. But on the other hand, `currentTime` and `position` are already the same anyway.

Reply:
Then again, there's lots of other metadata available from other events: https://docs.jwplayer.com/players/reference/metadata-events#onmeta

Maybe I should include the metadata as an unparsed `Value`? That way it's available if people want it.

Reply:
Resolved by 46d03db69b4d37b137524d39a579a19ea7f2789a.